### ISSUE TITLE
toast instead of alert

### ISSUE THREAD

**Issue by jibranabsarulislam** [It-s-Saturday/BillTracker#14]

**Comment by kaleemullah786** [comment]
hey, i want to work on it. can you provide more details

**Comment by jibranabsarulislam** [comment]
hey @kaleemullah786, thanks for finding our project! I'm sorry we don't have a readme yet. This project uses Node, React, Flask, and MongoDB (run locally). For this particular issue, please refer to https://github.com/It-s-Saturday/BillTracker/blob/46ae0db3b4a5d2b26599cac866972e4685d90c4d/frontend/src/components/Unshown.jsx#LL88C29-L88C29

Currently, when a paid off ('unshown') bill is listed, there is an option to "put it back". This issue revolves around the alert coming up to refresh the page. Instead, this should be converted to a toast (popup) that does not block the user's navigation while an alert would be open.

![image](https://github.com/It-s-Saturday/BillTracker/assets/70596906/1483d73d-301e-46b6-b128-2333ed69d4a4)

(all transactions and accounts in any screenshots are fictional and used as placeholder data)

**Comment by kaleemullah786** [comment]
so all we need to do is to convert this alert to toast. right
did you have any design and any suggestion about its position

**Comment by jibranabsarulislam** [comment]
nope, completely up to you. this is a pretty casual project so most things are fine no worries 🤝
